src/components/checkbox/Checkbox.js

```diff
@@ -177,6 +177,18 @@ export default class CheckBoxComponent extends Field {
     return false;
   }
 
+  updateValue(value, flags) {
+    const checkBoxChecked = 'checkbox-checked';
+    const val = this.getValue();
+    if (val) {
+      this.addClass(this.refs.formCheck, checkBoxChecked);
```

Comment:
Need to protect against templates that do not have this reference. It will crash otherwise.